pkgs/development/python-modules/tensorflow-probability/default.nix

```diff
@@ -117,6 +117,6 @@ in buildPythonPackage {
     description = "Library for probabilistic reasoning and statistical analysis";
     homepage = "https://www.tensorflow.org/probability/";
     license = licenses.asl20;
-    maintainers = with maintainers; [];  # This package is maintainerless.
+    maintainers = with maintainers; [ GaetanLepage ];
```

Comment:
Thanks for being the maintainer.
Could you please change this in another commit, `python311Packages.tensorflow-probability: add GaetanLepage as maintainers`?